src/test/TaskTest.java

```diff
@@ -0,0 +1,21 @@
+package Velma.task;
+
+import org.junit.jupiter.api.Test;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class TaskTest {
+
+    @Test
+    public void testChangeIsDone() {
```

Comment:
Could the test be better named according to the conventions?

Reply:
Yes the test name could have been named better.